assets/airports/ktus.json

```diff
@@ -0,0 +1,2422 @@
+        "SSAND": ["N32.15927955562581", "W110.33088683652029"],
+        "SSO"  : ["N32.26925215165929", "W109.26308440725370"],
+        "SULLI": ["N31.93437143337281", "W110.57125854744937"],
+        "SUVLI": ["N29.59527212245608", "W111.02966308506525"],
+        "TFD"  : ["N32.88587247171906", "W111.90872954902221"],
+        "TORTS": ["N32.23092499860011", "W111.18667602789657"],
+        "TUS"  : ["N32.09519837978357", "W110.91487884095169"],
+        "VAINE": ["N31.93549801560627", "W110.72804259826410"],
+        "VEYLE": ["N31.94097539856976", "W110.73758697082330"],
+        "VYLLA": ["N31.33346343830173", "W110.94982910068914"],
+        "WASON": ["N32.24619971978421", "W111.09237670473242"],
+        "WLDKT": ["N32.22436128955165", "W111.06703948549253"],
+        "ZONNA": ["N32.20925717812205", "W109.84865569635313"]
+    },
+    "runways":[
+        {
+            "name": ["3", "21"],
+            "end": [
+                ["N32d07m01.80", "W110d57m32.55", "2560ft"],
+                ["N32d07m50.74", "W110d56m34.96", "2569ft"]
+            ],
+            "ils": [false, false]
+        },
+        {
+            "name": ["11L", "29R"],
+            "end": [
+                ["N32d07m24.13", "W110d56m52.48", "2578ft"],
+                ["N32d06m07.17", "W110d55m22.15", "2643ft"]
+            ],
+            "ils": [true, true],
+            "glideslope": [3.00, 3.47]
+        },
+        {
+            "name": ["11R", "29L"],
+            "end": [
+                ["N32d07m19.57", "W110d56m58.75", "2574ft"],
+                ["N32d06m20.72", "W110d55m49.66", "2629ft"]
+            ],
+            "ils": [false, false]
+        }
+    ],
+    "sids": {
+        "BURRO4": {
+            "icao": "BURRO4",
+            "name": "Burro Four",
+            "rwy": {
+                "11L": []
+            },
+            "body": ["BURRO"],
+            "exitPoints": {
+                "BBALL": ["SPLTZ", ["RYYNN", "A80+"], "BBALL"],
+                "GBN": ["SPLTZ", ["RYYNN", "A80+"], ["ROSKR", "A90+"], "GBN"],
+                "HOTTT": ["SPLTZ", ["RYYNN", "A80+"], "HOTTT"],
+                "NOCHI": ["VEYLE", ["MESCA", "A95+"], "NOCHI"]
+            },
+            "draw": [
+                ["BURRO", "SPLTZ", "RYYNN", "ROSKR", "GBN*"],
+                ["BURRO", "VEYLE", "MESCA", "NOCHI*"],
+                ["RYYNN", "BBALL*"],
+                ["RYYNN", "HOTTT*"]
+            ]
+        },
+        "TUS8": {
+            "icao": "TUS8",
+            "name": "Tucson Eight",
+            "rwy": {
+                "3": [],
+                "11L": [],
+                "11R": [],
+                "21": [],
+                "29L": [],
+                "29R": []
+            },
+            "body": ["TUS"],
+            "exitPoints": {
+                "BAYBI": ["BAYBI"],
+                "BBALL": ["BBALL"],
+                "GBN": ["ROSKR", "GBN"],
+                "MESCA": ["MESCA", "SSO"],
```

Comment:
If necessary, we can just remove `SSO` from the exitpoints and add it at the end of the spawnpatterns